src/data/header.yaml

```diff
@@ -3,9 +3,9 @@ sticky: true
 shadow: true
 border: false
 headerPre:
-  title: "Stai usando l'anteprima del nuovo Designers Italia"
-  link: Il tuo feedback è prezioso
-  url: "/community/notizie/20221220-ecco-l-anteprima-del-nuovo-designers-italia"
+  showOn: prossima.designers.italia.it
+  title: Ciao, stai navigando una versione del sito non aggiornata e che verrà presto disattivata
```

Comment:
@bfabio @danielaiozzo a meaning doubt: the message is lightly untrue... you are browsing a version of the website that it is the latest indeed. It is "just" the URL that is old... and we want, with this banner, the user updates its bookmarks mainly, no?

Reply:
@Fupete that's right, good point

Reply:
uhm, so you mean the "prossima" version will be always in sync with the primary domain? I though "prossima" remained a sort of older version...

Reply:
@danielaiozzo that's correct, it will remain in sync with the main site as long as we keep it around.

([details here](https://github.com/italia/designers.italia.it/issues/653#issuecomment-1445037819))